inst/htmlwidgets/lib/profvis/profvis.js

```diff
@@ -1415,7 +1455,7 @@ profvis = (function() {
 
           // Size and position the panels
           $panel1.outerHeight($splitBar.position().top - splitBarGap.top -
-                             margin.top);
+                             margin.top - $statusBar.innerHeight());
```

Comment:
Does the positioning work for a vertical split?

I'm not sure if this was apparent, but the strategy I originally used for placement of the panels is this:

* Do a fixed layout with profvis.css.
* Call `initResizing`. This function takes the original positioning and moves/resizes components to a 50-50 proportion. It preserves the dimensions and spacing of various things.

So if you comment out the call to `initResizing()`, you can position all the elements with CSS. The layout should look roughly correct, though not correctly sized for a particular window height/width. Then when you uncomment `initResizing()`, that should position things correctly.